mentoring/pairing/rest.py

```diff
@@ -22,3 +22,11 @@ class Meta:
 class PairViewSet(mixins.CreateModelMixin, viewsets.ReadOnlyModelViewSet):
     queryset = Pair.objects.all()
     serializer_class = PairSerializer
+
+    def perform_create(self, serializer):
+        serializer.save()
+
+        # bump expiration for the participants involved
```

Comment:
Unrelated to this PR, but I realized it was necessary when doing a similar thing for participant updates.